src/Http/HttpAbstractions.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.26124.0
+# Visual Studio Version 16
```

Comment:
Does this still open in VS 2017? We haven't started requiring VS 2019 (yet)

Reply:
It does